File: golang/warmup/is_pangram_test.go
package warmup

import (
	"testing"

	"github.com/stretchr/testify/require"
)

type isPangramTest struct {
	sentence string
	expected bool
}

var isPangramTests = []isPangramTest{
	{
		sentence: "This is not a pangram",
		expected: false,
	},
	{
		sentence: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ",
		expected: true,
	},
}

func Test_isPangram(t *testing.T) {
	for _, tc := range isPangramTests {
		actual := isPangram(tc.sentence)

		require.Equal(t, tc.expected, actual)
	}
}

File: javascript/tests/warmup/isPangram.test.js
import { isPangram } from "../../src/warmup/isPangram";

describe("isPangram.js", () => {
    const tests = [
        {
            sentence: "This is not a pangram",
            expected: false,
        },
        {
            sentence: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ",
            expected: true,
        },
    ];

    tests.forEach((test, idx) => {
        it(`testCase: ${idx}`, () => {
            expect(isPangram(test.sentence)).toBe(test.expected);
        });
    });
});
